docs/geode-native-book/Gemfile.lock

```diff
@@ -202,4 +202,4 @@ DEPENDENCIES
   libv8 (= 3.16.14.7)
 
 BUNDLED WITH
-   1.16.1
+   1.11.2
```

Comment:
Why is this being downgraded?

Reply:
I had an older version of bundler installed, this will get updated in future docs stories.